### PR TITLE
re #39 append extra properties for CardPanel

### PR DESCRIPTION
**Pull request by isaiah**
@alextrastero

### CODE REVIEW

**Comment by alextrastero** [comment]
LGTM :+1:

File: src/CardPanel.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const CardPanel = ({
   };
 
   return (
-      <div className={cx(classes, className)}>
+      <div className={cx(classes, className)} {...props}>
         <span>{children}</span>
```
Comment (alextrastero) [inline src/CardPanel.js]: This will produce an error if `children` is/are block elements

**Comment by isaiah** [comment]
Do you mean we should remove the span element?

**Comment by isaiah** [comment]
Do you mean we should remove the span element?

**Comment by alextrastero** [comment]
Have you tried just adding {children} without a wrapper? If that doesn't work I would change `span` to `div`

**Comment by isaiah** [comment]
@alextrastero removed, I think it makes sense.

**Comment by isaiah** [comment]
@alextrastero If everything is ok, can you merge?

**Pull request merged**